src/AudioClip.cpp

```diff
@@ -1126,5 +1126,5 @@ uint64_t AudioClip::getCullImmunity() {
 	uint32_t distanceFromEnd = loopLength - getLivePos();
 	bool doingTimeStretching =
 	    (voiceSample && voiceSample->timeStretcher); // We're gonna cull time-stretching ones first
-	return ((uint64_t)voicePriority << 33) + ((uint32_t)!doingTimeStretching << 32) + distanceFromEnd;
+	return ((uint64_t)voicePriority << 33) + ((uint64_t)!doingTimeStretching << 32) + distanceFromEnd;
```

Comment:
This is a functional change but is probably a bugfix.

Reply:
Can you confirm for me if this piece of code still works as intended? I'm with you that it's probably a bug fix or oversight originally, but on the off chance it was intentional.

Reply:
I don't have a project around that would hit this code. Should I just rip this commit out to a separate PR so we can consider it in isolation? It'll be easy to test once I can set breakpoints.

Reply:
Probably safest to do that.